gnomad/sample_qc/relatedness.py

```diff
@@ -1524,30 +1528,30 @@ def default_get_de_novo_expr(
     med_conf_p: float = 0.5,
 ) -> hl.expr.StructExpression:
     """
-    Get the de novo status of a variant based on the proband and parent genotypes.
+    Get the *de novo* status of a variant based on the proband and parent genotypes.
 
     Thresholds:
 
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    |   Category     | P(de novo) | AB                    | AD   | DP  | DR   | GQ  |
-    +================+============+=======================+======+=====+======+=====+
-    | FAIL           | < 0.05     | AB(parents) > 0.05 OR |  0   |     | <0.1 | <20 |
-    |                |            | AB(proband) < 0.2     |      |     |      |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    | HIGH (Indel)   | > 0.99     | > 0.3                 |      |     |      |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    | HIGH (SNV) 1   | > 0.99     | > 0.3                 |      |     | >0.2 |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    | HIGH (SNV) 2   | > 0.5      | > 0.3                 |      | >10 |      |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    | MEDIUM         | > 0.5      | > 0.3                 |      |     |      |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
-    | LOW            | >= 0.05    | >= 0.2                |      |     |      |     |
-    +----------------+------------+-----------------------+------+-----+------+-----+
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    |   Category     | P(*de novo*) | AB                    | AD   | DP  | DR   | GQ  |
+    +================+==============+=======================+======+=====+======+=====+
+    | FAIL           | < 0.05       | AB(parents) > 0.05 OR |  0   |     | <0.1 | <20 |
+    |                |              | AB(proband) < 0.2     |      |     |      |     |
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    | HIGH (Indel)   | > 0.99       | > 0.3                 |      |     |      |     |
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    | HIGH (SNV) 1   | > 0.99       | > 0.3                 |      |     | >0.2 |     |
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    | HIGH (SNV) 2   | > 0.5        | > 0.3                 |      | >10 |      |     |
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    | MEDIUM         | > 0.5        | > 0.3                 |      |     |      |     |
+    +----------------+--------------+-----------------------+------+-----+------+-----+
+    | LOW            | >= 0.05      | >= 0.2                |      |     |      |     |
```

Comment:
just to revive the comment thread that became outdated -- I commented here that Kaitlin didn't use `>=`, and you were aware of this.

to respond to your question (`but where do those =0.05 and =0.2 belong?`): 
it doesn't seem like it would make a big difference to mark these variants as failing or to allow them to pass with low confidence. the more important piece is that this is well documented -- please add a note to the docstring that this is new behavior (this is a change from Kaitlin's original code)